src/modules/service-catalog/components/service-catalog-item/CollapsibleDescription.tsx

```diff
@@ -50,7 +50,15 @@ interface CollapsibleDescriptionProps {
   description: string;
 }
 
-const DESCRIPTION_LENGTH_THRESHOLD = 270;
+export const shouldShowToggleButton = (
+  description: string | null | undefined
+): boolean => {
+  const DESCRIPTION_LENGTH_THRESHOLD = 270;
+  return (
+    typeof description === "string" &&
```

Comment:
(nit) Maybe we can check for the presence of the description here instead of using typeof. I believe when you do something like 
```typescript
if (description) {
...
}
```

It returns false for both `null` and `undefined`. No strong feelings here though, since we know the exact type of the description.